src/main/scala/se/lu/nateko/cp/data/formats/delimitedheadercsv/DelimitedHeaderCsvStreams.scala

```diff
@@ -10,25 +10,46 @@ import java.time.Instant
 import se.lu.nateko.cp.meta.core.data.IngestionMetadataExtract
 import akka.stream.scaladsl.Keep
 import java.time.temporal.ChronoUnit
-import java.time.{ Instant, LocalDateTime, ZoneOffset }
+import java.time.{ Instant, LocalDateTime, LocalDate, LocalTime, ZoneOffset }
 import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
 
 object SitesDelimitedHeaderCsvStreams extends StandardCsvStreams {
+
 	import DelimitedHeaderCsvParser._
 
 	private val isoLikeDateFormater = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
 	private val columnSeparator = ","
 	private val headerDelimitor = "####"
+	private val formatter = new DateTimeFormatterBuilder()
+		.appendPattern("yyyy")
+		.parseDefaulting(ChronoField.MONTH_OF_YEAR, 1)
+		.parseDefaulting(ChronoField.DAY_OF_MONTH, 1)
+		.toFormatter();
 
 	def isNull(value: String, format: ValueFormat): Boolean = format match {
 		case FloatValue => value == "NaN" || value == "LOD" || value == ""
 		case _ => false
 	}
 
-	def makeTimeStamp(cells: Array[String]): Instant =
-		LocalDateTime.parse(cells(0), isoLikeDateFormater).toInstant(ZoneOffset.ofHours(1))
+	override def getTimestampValueFormat(colsMeta: ColumnsMeta): Option[ValueFormat] =
+		colsMeta.matchColumn("TIMESTAMP")
+	
+	def makeTimeStamp(cells: Array[String], timestampValueFormat: Option[ValueFormat]): Instant = timestampValueFormat match {
+		case Some(IsoLikeLocalDateTime) => LocalDateTime.parse(cells(0), isoLikeDateFormater).toInstant(ZoneOffset.ofHours(1))
+		case Some(Iso8601Date) => LocalDate.parse(cells(0)).atTime(LocalTime.MIN).toInstant(ZoneOffset.ofHours(1))
+		case Some(IntValue) => LocalDate.parse(cells(0), formatter).atTime(LocalTime.MIN).toInstant(ZoneOffset.ofHours(1))
+		case _ => Instant.EPOCH
+	}
 
 	def makeParser(format: ColumnsMetaWithTsCol): TextFormatParser =
 		new DelimitedHeaderCsvParser(format.colsMeta, columnSeparator, headerDelimitor)
 
+	override def acqIntervalTimeStep(timestampValueFormat: Option[ValueFormat]) = timestampValueFormat match {
```

Comment:
One year should still be handled, right? Need to also solve the problem of `Instant` not being incremented by years.